# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1207583437 14400
#      Mon Apr 07 11:50:37 2008 -0400
# Node ID b9f5f281b54be11bdc3660ba2f3bf966dfa6f922
# Parent  efccca5f2ad7ca8768fe86e982df71f485490f48
zchinx.f: use dble instead of real

diff --git a/libcruft/qrupdate/zchinx.f b/libcruft/qrupdate/zchinx.f
--- a/libcruft/qrupdate/zchinx.f
+++ b/libcruft/qrupdate/zchinx.f
@@ -37,21 +37,21 @@ c               definiteness.
       double complex R(n,n),R1(n+1,n+1),u(n+1)
       double precision rho,dznrm2
       double complex Qdum,w
       external zcopy,zlacpy,ztrsv,dznrm2
       integer jj
 
 c quick return if possible
       if (n == 0) then
-        if (real(u(1)) <= 0) then
+        if (dble(u(1)) <= 0) then
           info = 1
           return
         else
-          R(1,1) = sqrt(real(u(1)))
+          R(1,1) = sqrt(dble(u(1)))
         end if
       end if
 
 c check arguments      
       info = 0
       if (n < 0) then
         info = 1
       else if (j < 1 .or. j > n+1) then
